build/yamls/windows/base/conf/antrea-agent.conf

```diff
@@ -78,3 +78,8 @@ featureGates:
 # hybrid:            noEncap if source and destination Nodes are on the same subnet, otherwise encap.
 #
 #trafficEncapMode: encap
+
+# The name of the interface on Node which is used for tunneling or routing the traffic across Nodes.
+# If there are multiple IP addresses configured on the interface, the first one is used.
+# The interface configured with Node IP is used if this configuration is not set.
```

Comment:
"this configuration" -> "this parameter"

Reply:
updated.